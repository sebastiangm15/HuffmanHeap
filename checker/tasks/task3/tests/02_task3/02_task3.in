8
6 SATJX
27 SAT1A
16 SATYG
17 SAT5A
20 SATG2
7 SAT8K
25 SATHU
7 SATYD
5
SAT5A
SAT8K
SATG2
SAT1A
SATJX
