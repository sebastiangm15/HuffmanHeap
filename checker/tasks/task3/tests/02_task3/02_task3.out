101111111100111110
